easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-foss-2016a.eb

```diff
@@ -34,6 +42,9 @@ dependencies = [
     ('GST-plugins-base', '1.6.4'),
 ]
 
+#The destination directory is "linux-<MD5>" see 3rdparty/ippicv/downloader.cmake
+preconfigopts = 'mkdir -p {0} && cp -a ../*.tgz {0}. &&'.format("3rdparty/ippicv/downloads/linux-808b791a6eac9ed78d32a7666804320e/")
```

Comment:
@hajgato line is too long, how about this:

```python
ippicv_dir = '3rdparty/ippicv/downloads/linux-808b791a6eac9ed78d32a7666804320e/'
preconfigopts = "mkdir -p {0} && cp -a ../*.tgz {0} && ".format(ippicv_dir)
```

I'm not a big fan of using `.format` in easyconfigs, but admittedly this is not better:

```python
preconfigopts = "mkdir -p %s && cp -a ../*.tgz %s && " % (ippicv_dir, ippicv_dir)
```

That does follow the easyconfigs style we usually go with though...